### PR TITLE
Added JSON file and inline content-disposition

### PR DESCRIPTION
**Pull request by prioux**
This add support for JSON files in CBRAIN.

Also both XML and JSON files can be viewed in the browser's own viewer (Firefox supports both, Chrome only seems to support XML).

### CODE REVIEW

**Comment by prioux** [comment]
This will fix #866 by the way.

File: BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/json_file/views/_json_raw.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<p>
+The link below will open this JSON file directly in your browser.
+<p>
+Some modern browsers have a built-in JSON viewer.
+<p>
+
+<%= link_to "Browser-based viewer for JSON content",
```
Comment (natacha-beck) [inline BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/json_file/views/_json_raw.html.erb]: I personally prefer that the link opens in a new tab, and it is what it´s done for raw Text file for example. We should decide which behaviour is better.

File: BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/xml_file/views/_xml_raw.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<p>
+The link below will open this XML file directly in your browser.
+<p>
+Some modern browsers have a built-in XML viewer.
+<p>
+
+<%= link_to "Browser-based viewer for XML content",
```
Comment (natacha-beck) [inline BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/xml_file/views/_xml_raw.html.erb]: Same comment as before regarding, maybe it should open in a new tab.

File: BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/json_file/views/_json_raw.html.erb (hunk 1 above)
Comment (prioux) [reply BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/json_file/views/_json_raw.html.erb]: I didn't think of it, but it's a good idea. It should be easy to do...

Comment (natacha-beck) [reply BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/json_file/views/_json_raw.html.erb]: yes  `:target => "_blank"` and that's it.

Comment (prioux) [reply BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/json_file/views/_json_raw.html.erb]: I'll do it tomorrow.

**Comment by prioux** [comment]
@natacha-beck Added the target=_blank; Travis CI passed.

**Pull request merged**